app/views/sephora_style_guide/home/_svg_icons.slim

```diff
@@ -1,22 +1,90 @@
+    { "name": "Search Black", "class": "search-black" }, { "name": "Search White", "class": "search-white" }, { "name": "Scan", "class": "scan" }, { "name": "Wishlist Circle Black", "class": "heart-empty-bg" },
+    { "name": "Wishlist Circle Red", "class": "heart-empty-red-bg" }, { "name": "Wishlisted Circle", "class": "heart-fill-bg" } ]
+  - social_media = [ { "name": "Instagram", "class": "instagram" }, { "name": "Snapchat", "class": "snapchat" }, { "name": "Youtube", "class": "youtube" }, { "name": "Twitter", "class": "twitter" },
+    { "name": "Google", "class": "google" }, { "name": "Facebook", "class": "facebook" }, { "name": "Email", "class": "email" }, { "name": "Pinterest", "class": "pinterest" } ]
+  - others = [ { "name": "Lipstick Rated", "class": "lipstick-filled" }, { "name": "Lipstick Unrated", "class": "lipstick-empty" }, { "name": "Print", "class": "print" }, { "name": "Save as PDF", "class": "pdf" },
+    { "name": "Samples", "class": "makeup-2" }, { "name": "No Filter Found", "class": "makeup-1" }, { "name": "Triangle", "class": "triangle" }, { "name": "Triangle Black", "class": "triangle-black" },
+    { "name": "Size", "class": "size" }, { "name": "Show", "class": "show-icon" }, { "name": "Shade", "class": "shade" }, { "name": "Triangle Black", "class": "triangle-black" }, { "name": "Processing", "class": "processing" },
+    { "name": "Field Error", "class": "field-error" }, { "name": "Field Success", "class": "field-succeed" }, { "name": "Radio Disabled Selected", "class": "radio-disabled-selected" }, { "name": "Radio Disabled Unselected", "class": "radio-disabled-unselected" },
+    { "name": "Radio Selected", "class": "radio-selected" }, { "name": "Radio Unselected", "class": "radio-unselected" }, { "name": "Selected Option", "class": "selected-option-icon" },
+    { "name": "Giftcard", "class": "giftcard" }, { "name": "Location", "class": "location" } ]
 
   section
-    h3.h3 Icons
-    - svg_icons.each do | icon |
-      .icon_container
-        .svg-icon class="#{ icon }"
-        span #{icon}
+    h3.h3 Navigation and Menu
+    - navigation_icons.each do | icon |
+      .icon_container.col-12.col-md-6.col-lg-4
+        .helper
+        .svg-icon class="#{ icon[:class] }"
+        div.icon-data
+          p #{ icon[:name] }
+          p.subtext .svg-icon.#{ icon[:class] }
+    h3.h3 Orders and Shipping Status
+    - orders_and_shipping.each do | icon |
+      .icon_container.col-12.col-md-6.col-lg-4
+        .helper
+        .svg-icon class="#{ icon[:class] }"
+        div.icon-data
+          p #{ icon[:name] }
+          p.subtext .svg-icon.#{ icon[:class] }
+    h3.h3 My Account
+    - my_account.each do | icon |
+      .icon_container.col-12.col-md-6.col-lg-4
+        .helper
+        .svg-icon class="#{ icon[:class] }"
+        div.icon-data
+          p #{ icon[:name] }
```

Comment:
If it's just a single argument that needs to be evaluated as ruby, you can do `p= icon[:name]` (Can't recall if there is a space between the `p` and the `=`)